Dockerfile

```diff
@@ -73,13 +74,13 @@ RUN curl -sL https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz | tar xz
     rm -rf /chruby-* && rm -rf /usr/local/src/* && yum clean all
 
 ## GIT clone manageiq-appliance and self-service UI repo (SSUI)
-RUN git clone --depth 1 https://github.com/ManageIQ/manageiq-appliance.git ${APPLIANCE_ROOT} && \
-git clone --depth 1 https://github.com/ManageIQ/manageiq-ui-self_service.git ${SSUI_ROOT} && \
+RUN git clone --depth 1 --branch ${REF} https://github.com/ManageIQ/manageiq-appliance.git ${APPLIANCE_ROOT} && \
+git clone --depth 1 --branch ${REF} https://github.com/ManageIQ/manageiq-ui-self_service.git ${SSUI_ROOT} && \
 ln -vs ${APP_ROOT} /opt/manageiq/manageiq
 
 ## Create approot, ADD miq
 RUN mkdir -p ${APP_ROOT}
-ADD . ${APP_ROOT}
+RUN git clone --depth 1 --branch ${REF} https://github.com/ManageIQ/manageiq.git ${APP_ROOT}
```

Comment:
I don't think this should go in.
The Dockerfile in ManageIQ repo should not clone manageiq repo for the following reasons:

1.  size (as stated above by Franco)
2. a developer looses its ability to build & test unmerged  code.

I would not add this commit to the upstream repo